src/Aspire.Hosting.AWS/Lambda/LambdaLifecycleHook.cs

```diff
@@ -22,10 +21,51 @@ internal class LambdaLifecycleHook(ILogger<LambdaEmulatorResource> logger, IProc
 
     public async Task BeforeStartAsync(DistributedApplicationModel appModel, CancellationToken cancellationToken = default)
     {
+        var classLibraryProjectPaths = 
+            appModel.Resources
+                .OfType<LambdaProjectResource>()
+                .Where(x =>
+                {
+                    if (!x.TryGetLastAnnotation<LambdaFunctionAnnotation>(out var functionAnnotation))
```

Comment:
what is the significance of using the last annotation? i understand this whole thing is supposed to filter for class library project paths but wasnt clear on this one line

Reply:
i dont know the code well enough but is there any case where doing last vs first would have a different result?

Reply:
No they're the same. We only have one annotation in there. If for any reason a new one was added, taking the last one makes more sense than the first one.